### DISCUSSION TITLE
The ability to deploy a Linux from a Visual Studio Project

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7566] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/The-ability-to-deploy-a-Linux-from-a-Vis/10263584)._

---
It would be nice to have a deployment wizard to select a target Linux/Windows machine and then copy all the necessary files, libraries, environment, etc.

---
#### Feedback Bot on 1/29/2023, 00:54 AM: 

(private comment, text removed)
#### Sy Brand [MSFT] on 2/15/2023, 09:35 AM: 

(private comment, text removed)
#### Feedback Bot on 2/22/2023, 00:11 AM: 

(private comment, text removed)
#### David Blanco on 2/28/2023, 02:11 AM: 

(private comment, text removed)
#### Sy Brand [MSFT] on 2/28/2023, 03:27 AM: 

(private comment, text removed)
#### Sy Brand [MSFT] on 2/28/2023, 03:31 AM: 

(private comment, text removed)

**Comment by cwebster-99** [comment]
Additional Info:

"Sorry, I forgot to mention that this is about Python Projects.
It would be nice to have a wizard like Web Projects have.
WSL doesn’t fit for me because they’re remote devices."
